docs/source/input_preparation.rst

```diff
@@ -0,0 +1,115 @@
+Instructions for preparing inputs for the OpenFE public benchmark
+#################################################################
+
+Overview & Aims
+***************
+
+This document provides guidance on preparing the `Schrodinger benchmark files <https://github.com/schrodinger/public_binding_free_energy_benchmark/tree/v2.0/fep_benchmark_inputs/structure_inputs>`_ for use in the OpenFE 2024 public dataset industry benchmark.
+
+Whilst each partner organization will be responsible for preparing inputs using their own tooling, this guidance aims to provide a set of rules to standardize the process and ensure that scientifically equivalent simulation inputs are generated.
+
+This document will be continually updated based on feedback from benchmark partners. Please reach out should you have any questions or require additional information whilst preparing your inputs.
+
+Input preparation instructions
+******************************
+
+Input data source
+=================
+
+All input data are sourced from the `v2.0 release of the Schrodinger free energy benchmark <https://github.com/schrodinger/public_binding_free_energy_benchmark/tree/v2.0>`_. Specifically, input PDB, SDF and edges CSV files can be found under the relevant `structure input <https://github.com/schrodinger/public_binding_free_energy_benchmark/tree/v2.0/fep_benchmark_inputs/structure_inputs>`_ sub-folder.
```

Comment:
Probably worth removing the second sentence here and mentioning the PDB, SDF, and CSV files below.